src/TransitionReplace/TransitionReplace.test.jsx

```diff
@@ -0,0 +1,80 @@
+/* eslint-disable no-plusplus, react/prop-types */
```

Comment:
Can we just take no-plusplus out of our linting?  It is not one of the rules I think is good.

Reply:
I agree. Tagged you over here: https://github.com/edx/eslint-config-edx/issues/47

Reply:
I thought it leads to bugs in places? Seems simple to replace with `x += 1`.

Reply:
++ on this choice